contracts/dtoken/src/borrow.rs

```diff
@@ -1,23 +1,25 @@
 use crate::*;
 
+const MIN_BORROW_GAS_MULTIPLIER: u64 = 120;
```

Comment:
@dariacherniak let's make those constant like that `GAS_FOR_BORROW: Gas = 120 * TGAS`